### ISSUE TITLE
Option to ignore the error specifically from the PR already existing

### ISSUE THREAD

**Issue by tscizzlebg** [vsoch/pull-request-action#50]
Hi there! Title says it mostly, but to elaborate:

There is a way to not fail the action if an error occurs (`PASS_ON_ERROR`). However, the most common failed check I see with this action is that "Validation Failed" due to "A pull request already exists for [head branch]". And we don't really consider that an error. If the desired PR already exists, and will incorporate new changes pushed to the head branch, then that's fine. I'm inclined to use `PASS_ON_ERROR` for now, but wish that it didn't catch all errors, since presumably an actual problem could arise and we might not realize.

So in summary, something like `PASS_IF_ALREADY_EXISTS` so that we could ignore this one specific case of "error" without ignoring all errors.

Lemme know your thoughts!

Best, tscizzlebg

**Comment by vsoch** [comment]
It's a great idea! Here is a branch to test. https://github.com/vsoch/pull-request-action/pull/51

**Comment by tscizzlebg** [comment]
Do you know how I can use that branch's version to test it out?

My workflow rn has the line: `uses: vsoch/pull-request-action@1.0.6`. Perhaps I can change something there to point to a branch?

**Comment by vsoch** [comment]
Yes that's exactly correct! You would change it to be the branch name.
```
vsoch/pull-request-action@add/pr-exists-error
```
And then the environment variable to set is `PASS_IF_EXISTS`
